crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -84,6 +84,28 @@ impl Name {
         }
     }
 
+    /// For a given Name generate the Rust and C++ wrapper names
+    pub fn wrapper_from(self) -> Self {
```

Comment:
as you are cloning things internally anyway you could change this to be `(&self)` so then you don't need another clone at the call site.

Reply:
done